server_installation/topics/clustering/sticky-sessions.adoc

```diff
@@ -31,11 +31,39 @@ and hence Infinispan can lookup this session locally. After authentication is fi
 The sticky session is not mandatory for the cluster setup, however it is good for performance for the reasons mentioned above. You need to configure your loadbalancer to sticky
 over the `AUTH_SESSION_ID` cookie. How exactly do this is dependent on your loadbalancer.
 
-Some loadbalancers can be configured to add the route information by themselves instead of rely on the backend {project_name} node. However adding the route by
-the {project_name} is more clever as {project_name} knows, who is the owner of particular session entity and can route it to that node, which is not necessarily the local node.
-We plan to support the setup where the route info won't be added to AUTH_SESSION_ID cookie by the {project_name} server, but instead by the load balancer. However
-adding the cookie by {project_name} would be still the preferred option.
+The recommended step on {project_name} side is to use the system property `jboss.node.name` during startup with the value corresponding to the name of your route.
+For example `-Djboss.node.name=node1` will use the `node1` as the identification of the route. This route will be used by Infinispan caches and will be
+ attached to the AUTH_SESSION_ID cookie in case that our node is the owner of the particular key. The example of the startup command with this system property is below in the
+ <<_example-setup-with-mod-cluster,Mod Cluster Example>> section.
+
```

Comment:
It is recommended on the {project_name} side to use the system property `jboss.node.name` during startup, with the value corresponding to the name of your route. For example, `-Djboss.node.name=node1` will use `node1` to identify the route. This route will be used by Infinispan caches and will be attached to the AUTH_SESSION_ID cookie when the node is the owner of the particular key. An example of the start up command with this system property can be seen in <<_example-setup-with-mod-cluster,Mod Cluster Example>>.